src/utils/winlnkfileparse.h

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-FileCopyrightText: 2017-2019 Alejandro Sirgo Rica & Contributors
+
+#pragma once
+
+#include "desktopfileparse.h"
+#include <QFileInfo>
+#include <QIcon>
+#include <QMap>
+#include <QStringList>
+
+class QDir;
+class QString;
+
+struct CompareAppByName {
+    bool operator()(const DesktopAppData a, const DesktopAppData b) {
+        return(a.name < b.name);
+    }
+};
+
+struct WinLnkFileParser
+{
+    WinLnkFileParser();
+    DesktopAppData parseLnkFile(const QFileInfo& fiLnk, bool& ok) const;
+    int processDirectory(const QDir& dir);
+    QString getAllUsersStartMenuPath();
+
+    QVector<DesktopAppData> getAppsByCategory(const QString& category);
+    QMap<QString, QVector<DesktopAppData>> getAppsByCategory(
+      const QStringList& categories);
+
+private:
+    void getImageFileextAssociates(const QStringList &sListImgExt);
```

Comment:
Probably should read getImageFileExtAssociates (capitalize E in Ext)

Reply:
:+1: 
I will change it + proper clang-format soon